src/elements/ts-main/ts-academy.html

```diff
@@ -102,18 +102,20 @@
 
             App.ipc.send('loading-status', 'Loading Academy data...');
             mythis.set('academy', App.dataManager.getAllTa());
+            var dataserver = App.configurator.getUserSetting('dataserver');
+            var mediaserver = App.configurator.getUserSetting('mediaserver');
 
             setTimeout(function() {
                 var links = mythis.$.body.getElementsByTagName("a");
                 var images = mythis.$.body.getElementsByTagName("img");
                 var imageDir = path.resolve(path.join(__dirname, '..', 'views', 'assets', 'img', 'ta'));
 
                 for (var j = 0; j < images.length; j++) {
-                    images[j].src = images[j].src.replace("http://cdn.door43.org/ta/jpg", imageDir);
+                    images[j].src = images[j].src.replace(mediaserver + "/ta/jpg", imageDir);
```

Comment:
So you actually might want to add a third thing to your configuration for cdn server because the media server is not the same as the cdn

Reply:
What is the difference between the two?

Reply:
It looks like this issue is the one snag?

Reply:
@PurpleGuitar The CDN contains files. The api is endpoints for functions. This link will end in a 404. I can make it work with a pass though in the api gateway but api.door43.org != cdn.door43.org.

Reply:
Since this is replacing this might be an error to change it. Unless the image that is being updated also is being generated from media server. But other then additional web traffic I don't think there is anything wrong with this. But probably should be reverted.

Reply:
Thanks, I agree.  I've reverted the change for now.